testsuite/features/init_clients/proxy_register_as_minion_with_script.feature

```diff
@@ -22,15 +22,15 @@ Feature: Setup Uyuni proxy
 
 @skip_if_salt_bundle
   Scenario: Create the bootstrap script for the proxy and use it
-    When I execute mgr-bootstrap "--script=bootstrap-proxy.sh"
+    When I execute mgr-bootstrap "--activation-keys=1-PROXY-KEY-x86_64 --script=bootstrap-proxy.sh"
     Then I should get "* bootstrap script (written):"
     And I should get "    '/srv/www/htdocs/pub/bootstrap/bootstrap-proxy.sh'"
     When I fetch "pub/bootstrap/bootstrap-proxy.sh" to "proxy"
     And I run "sh ./bootstrap-proxy.sh" on "proxy"
 
 @salt_bundle
   Scenario: Create the bundle-aware bootstrap script for the proxy and use it
-    When I execute mgr-bootstrap "--script=bootstrap-proxy.sh --force-bundle"
+    When I execute mgr-bootstrap "--activation-keys=1-PROXY-KEY-x86_64 --script=bootstrap-proxy.sh"
```

Comment:
We are making this scenario Architecture dependent. For now, no problem, as we only support SUSE Manager Proxy with X86_64, but I bet we will be supporting Proxy with ARM at some point soon.

Reply:
FMPOV it was architecture dependent before where we had the architecture hard coded in the step. With my changes, you are able to provide the activation key you want regardless of the architecture. The proxy key I created here is simply only for x86_64.